### PR TITLE
Fix CI for RHEL9

### PR DESCRIPTION
**Pull request by samjwu**

### CODE REVIEW

File: Tensile/Source/client/CMakeLists.txt [hunk 1]
```diff
@@ -47,6 +47,9 @@ if(NOT WIN32)
     )
 endif()
 
+add_definitions(-U_GLIBCXX_ASSERTIONS)
+add_definitions(-U_LIBCPP_ENABLE_ASSERTIONS)
+
```
Comment (ellosel) [inline Tensile/Source/client/CMakeLists.txt:52]: I don't think this is making any difference.

**Comment by TorreZuk** [comment]
Not sure if for the client the msgpack and LLVM should be exclusive?   i.e. if using msgpack there is no use of LLVM.  That is how we package rocBLAS but not sure how this client compares to our use.  Would be good to back out any changes that aren't required.

File: Tensile/Source/client/CMakeLists.txt (hunk 1 above)
Comment (samjwu) [reply Tensile/Source/client/CMakeLists.txt]: Removed in [Address comments](https://github.com/ROCm/Tensile/pull/1945/commits/ca90a4d3f085efd122989e729f2e0d803335a7fc)

File: .jenkins/common.groovy [hunk 2]
```diff
@@ -55,9 +61,12 @@ def runCompileCommand(platform, project, jobName, boolean debug=false)
             export TENSILE_COMPILER=${compiler}
             export HIPCC_COMPILE_FLAGS_APPEND='-O3 -Wno-format-nonliteral -parallel-jobs=4'
 
+            ${sclCommand}
+
             mkdir build && pushd build
 
             cmake ../HostLibraryTests \
+                -DTENSILE_USE_LLVM=OFF \
```
Comment (ellosel) [inline .jenkins/common.groovy:69]: We should try turning this back on no?

File: HostLibraryTests/CMakeLists.txt [hunk 3]
```diff
@@ -41,9 +41,7 @@ option(TENSILE_DISABLE_CTEST "Disable CTest integration." ON)
 
 if(TENSILE_USE_HIP)
     set(TENSILE_COMPONENTS HIP)
-    if(TENSILE_USE_LLVM)
-        set(TENSILE_COMPONENTS ${TENSILE_COMPONENTS} Client)
-    endif()
```
Comment (ellosel) [inline HostLibraryTests/CMakeLists.txt:46]: We can revert this now.

File: Tensile/ClientExecutable.py [hunk 4]
```diff
@@ -83,8 +83,8 @@ def clientExecutableEnvironment(builddir=None):
     CCompiler   = "clang.exe"   if ((os.name == "nt") and supportedCompiler(globalParameters['CxxCompiler'])) else globalParameters['CCompiler']
 
     options = {'CMAKE_BUILD_TYPE': globalParameters["CMakeBuildType"],
-               'TENSILE_USE_MSGPACK': 'ON',
-               'TENSILE_USE_LLVM': 'OFF' if (os.name == "nt") else 'ON',
```
Comment (ellosel) [inline Tensile/ClientExecutable.py:87]: and revert this

File: Tensile/cmake/TensileConfig.cmake [hunk 5]
```diff
@@ -68,7 +68,7 @@ if("Client" IN_LIST Tensile_FIND_COMPONENTS)
         set(TENSILE_BUILD_CLIENT ON CACHE BOOL "Build Client")
     elseif(Tensile_FIND_REQUIRED_Client)
         message("Tensile client requires both Hip and LLVM.")
-        set(Tensile_FOUND false)
+        #set(Tensile_FOUND false)
```
Comment (ellosel) [inline Tensile/cmake/TensileConfig.cmake:71]: and revert this

**Comment by samjwu** [comment]
CI runs are passing now on various pipelines. I am thinking this PR can be closed

http://math-ci.amd.com/job/hipclang-bleeding-edge/job/precheckin/job/Tensile/job/develop/46/pipeline-graph/

http://math-ci.amd.com/job/gfx94x/job/precheckin/job/Tensile/job/develop/54/pipeline-graph/

**Pull request closed without merging**